app/front/scripts/controllers/main.js

```diff
@@ -69,7 +73,24 @@ angular.module('Application')
       function initRegular() {
         $q(LoginService.tryGetToken())
           .then(function(token) {
-            return $q(osViewerService.loadDataPackages(token));
+            var packageId = null;
+            var userId = token ? LoginService.getUserId() : null;
+            if (!token) {
```

Comment:
'getInitialState` (in short) takes list of loaded packages (content of this list depends on algorithm described in the feature), and package ID from URL. If package with that ID exists in the list - this package is set as current; otherwise, first package from the list will be used as current.